Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -57,29 +69,29 @@ struct CreateBuffersAroundPointsView: View {
                     .disabled(model.bufferPoints.isEmpty)
                 }
             }
-            .alert("Buffer Radius", isPresented: $model.inputBoxIsPresented, actions: {
-                TextField("100", text: $model.radiusInput)
+            .alert("Buffer Radius", isPresented: $inputBoxIsPresented, actions: {
+                TextField("100", text: $radiusInput)
                     .keyboardType(.numberPad)
                 Button("Done") {
                     // Check to ensure the tapPoint is within the boundary.
-                    guard model.boundaryContains(model.tapPoint) else {
+                    guard let tapPoint, model.boundaryContains(tapPoint) else {
                         status = .outOfBoundsTap
                         return
                     }
                     
                     // Ensure that the input is valid.
-                    if let radius = Double(model.radiusInput),
+                    if let radius = Double(radiusInput),
                        radius > 0 && radius < 300 {
-                        model.addBufferPoint()
-                        model.drawBuffers()
+                        model.addBuffer(point: tapPoint, radius: radius)
+                        model.drawBuffers(shouldUnion: shouldUnion)
                         status = .bufferCreated
                     } else {
                         status = .invalidInput
                     }
-                    model.radiusInput = ""
+                    radiusInput.removeAll()
                 }
                 Button("Cancel") {
```

Comment:
Setting a role here allows the return key to activate the Done button.

```suggestion
                Button("Cancel", role: .cancel) {
```